src/options/modules/CustomOptionTriggers.js

```diff
@@ -17,7 +17,24 @@ import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunic
  * @param  {Object} [event]
  * @returns {Promise}
  */
-function applyAutocorrectPermissions(optionValue) {
+function applyAutocorrectPermissions(optionValue, option, event) {
+    if (optionValue.enabled) {
+        if (option && event?.target?.name === "enabled") {
```

Comment:
Did you make sure the minimum version in the `manifest.json` for Firefox we declare supports this `?.` syntax?

If not, please just bump the minimum version… :upside_down_face:

Reply:
Yes, the minimum version is Firefox 87:
https://github.com/rugk/unicodify/blob/65f5e4bab0adefa33436ef359e693913ebe7e12c/src/manifest.json#L47
and the optional chaining operator only requires Firefox 74 (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility)).